akka-discovery/src/main/scala/akka/discovery/ServiceDiscovery.scala

```diff
@@ -299,7 +308,7 @@ abstract class ServiceDiscovery {
    * the passed `resolveTimeout` should never be exceeded, as it signals the application's
    * eagerness to wait for a result for this specific lookup.
    *
-   * The returned future SHOULD be failed once resolveTimeout has passed.
+   * The returned future should be failed once resolveTimeout has passed with a [DiscoveryTimeoutException].
```

Comment:
does it work with single `[  ]`? I think we use double in most places

Reply:
:+1: doh